### PR TITLE
Training sections

### PR DESCRIPTION
**Pull request by harshahemanth**
categorising into sections available, locked, in-progress and completed

### CODE REVIEW

File: safety-quiz/userflow.py [hunk 1]
```diff
@@ -1,18 +1,68 @@
...
+
+    for i in locked_query:
+
+        search_if_complete = db.query(Machine).outerjoin(Training).filter(Training.trainee_id == session['sid']) \
+            .filter(Machine.parent_id != None).filter(Training.machine_id == i.parent_id) \
+            .filter(Training.quiz_score != passing_score).all()
+        if search_if_complete != None:
+            locked_list.append(i)
+        else:
+            available_list.append(i)
```
Comment (pkarnia) [inline safety-quiz/userflow.py:65]: When I test this function I get the same items in locked and completed.

**Review by pkarnia — CHANGES_REQUESTED** [review]
I will review / test again once these changes have been made.

File: safety-quiz/userflow.py [hunk 2]
```diff
@@ -1,18 +1,68 @@
...
+    locked_list = []
+    available_list = []
+
+    # For Completed and In-progress
+    overall_training = db.query(Training).outerjoin(Machine).filter(Training.trainee_id == session['sid']) \
+        .filter(Training.quiz_score == passing_score).filter(Training.invalidation_date == None) \
+        .filter(Training.in_person_date is not None).all()
+
+    for training in overall_training:
+        if Training.completed(training):
```
[the remark this answers is not in the document]
Comment (harshahemanth) [reply safety-quiz/userflow.py:47]: No, I'm looping though the ovarall_list and checking with completed function, 
"for training in overall_training"

[the remark this answers is not in the document]
Comment (harshahemanth) [reply safety-quiz/userflow.py]: Changed

**Comment by harshahemanth** [comment]
Creating another PR

**Pull request closed without merging**